src/components/inputs/toggle-input/toggle-input.styles.js

```diff
@@ -1,185 +1,138 @@
-import { css } from '@emotion/core';
+import styled from '@emotion/styled';
 import vars from '../../../../materials/custom-properties';
 
-const toggleSwitchSizes = {
-  small: {
-    height: '16px',
-    width: '32px',
-  },
-  big: {
-    height: '32px',
-    width: '64px',
-  },
-};
 const toggleButtonSizes = {
   small: {
-    height: '16px',
-    width: '16px',
+    height: '13px',
+    width: '13px',
   },
   big: {
-    height: '32px',
-    width: '32px',
+    height: '26px',
+    width: '26px',
   },
 };
+
 const toggleBarSizes = {
   small: {
-    height: '12px',
+    height: '16px',
     width: '32px',
   },
   big: {
-    height: '24px',
+    height: '32px',
     width: '64px',
   },
 };
 
-const getToggleSwitchStyles = props => {
-  const baseStyles = [
-    css`
-      &:hover > * {
-        filter: drop-shadow(${vars.shadow15First})
-          drop-shadow(${vars.shadow15Second});
-      }
-    `,
-    props.isDisabled &&
-      css`
-        &:hover > * {
-          filter: none;
-        }
-      `,
-  ];
-  switch (props.size) {
-    case 'small':
-      return [
-        ...baseStyles,
-        css`
-          position: relative;
-          height: ${toggleSwitchSizes.small.height};
-          width: ${toggleSwitchSizes.small.width};
-        `,
-      ];
-    case 'big':
-      return [
-        ...baseStyles,
-        css`
-          position: relative;
-          height: ${toggleSwitchSizes.big.height};
-          width: ${toggleSwitchSizes.big.width};
-        `,
-      ];
-    default:
-      return baseStyles;
-  }
-};
+export const Button = styled.div`
+  position: absolute;
 
-const getToggleButtonStyles = props => {
-  const baseStyles = [
-    css`
-      position: absolute;
-      z-index: 1;
-      filter: drop-shadow(${vars.shadow11First})
-        drop-shadow(${vars.shadow11Second});
-      transition: ${vars.transitionStandard};
-
-      svg .bt-fill {
-        transition: ${vars.transitionStandard};
-      }
-    `,
+  transform: translateY(-50%);
+  top: 50%;
+
+  background-color: ${vars.colorSurface};
+  box-shadow: ${vars.shadow7};
+  border-radius: 50%;
+  z-index: 1;
+  transition: transform ${vars.transitionStandard};
```

Comment:
This doesn't work, because `transitionStandard` already has a property name (`all`), resulting in an invalid rule. So it should just be `transform`.

Not related to this PR, but `transitionStandard` shouldn't have the property specified, and especially not `all`, as it's a very bad practice because of how it can hurt performance.